mappings/net/minecraft/world/biome/layer/samplingUtil/Sampler1.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS chr net/minecraft/world/biome/layer/samplingUtil/Sampler1
```

Comment:
Shouldn't these have better names than just `Sampler1`, `Sampler2`, etc.?

Reply:
There are definitely better ways to name them. Consider the pattern created by the sampled pixels.